packages/integrations/vercel/types.d.ts

```diff
@@ -1,5 +1,9 @@
+import { waitUntil } from '@vercel/functions';
+
 export interface EdgeLocals {
 	vercel: {
-		edge: import('@vercel/edge').RequestContext;
+		edge: {
+			waitUntil: typeof waitUntil;
```

Comment:
I am not familiar with vercel, but is ˋwaitUntilˋ everything from the old ˋRequestContextˋ ?

Reply:
Yes, it was. Technically the package exports more functions, but I wasn't sure if and how we document the `EdgeLocals`, hence I didn't want to add new stuff